### ISSUE TITLE
Swipe gesture on left menu disallow left to right gesture

### ISSUE THREAD

**Issue by shaharukhs** [jonkykong/SideMenu#271]
## New Issue Checklist

I have read the [guidelines for contributing](https://github.com/jonkykong/SideMenu/blob/master/.github/CONTRIBUTING.md) and I understand:
- [x] My issue is happening in the **latest version** of SideMenu.
- [x] My issue was **not** solved in the [README](https://github.com/jonkykong/SideMenu/blob/master/README.md).
- [ ] My issue can **not** be answered on [stackoverflow.com](stackoverflow.com).
- [x] My issue is **not** a request for new functionality that I am unwilling to build and contribute with a pull request.
- [ ] My issue **is** reproducible in the [demo project](https://github.com/jonkykong/SideMenu/tree/master/Example).

## Issue Description

I have implemented side menu as follow

```
// Define the menus
let sideMenuObj : SideMenuViewController! = SideMenuViewController(nibName: "SideMenuViewController", bundle: nil)
let menuLeftNavigationController = UISideMenuNavigationController(rootViewController: sideMenuObj)
menuLeftNavigationController.leftSide = true
SideMenuManager.menuLeftNavigationController = menuLeftNavigationController
// Enable gestures. The left and/or right menus must be set up above for these to work.
// Note that these continue to work on the Navigation Controller independent of the view controller it displays!
SideMenuManager.menuAddPanGestureToPresent(toView: view)
SideMenuManager.menuAddScreenEdgePanGesturesToPresent(toView: view)
		
SideMenuManager.menuWidth = UIScreen.main.bounds.size.width
SideMenuManager.menuDismissOnPush = true
SideMenuManager.menuEnableSwipeGestures = false
SideMenuManager.menuAllowPushOfSameClassTwice = false
```

But when I swipe gesture on my view from right to left side menu still open but I want only gesture for right to left. So ow can I disable left to right gesture.

Thank you.

**Comment by jonkykong** [comment]
`menuAddScreenEdgePanGesturesToPresent(toView:forMenu:)`.

**Comment by shaharukhs** [comment]
I don't understand `forMenu:`

**Comment by jonkykong** [comment]
Please check the README.

**Comment by shaharukhs** [comment]
`SideMenuManager.menuAddScreenEdgePanGesturesToPresent(toView: view, forMenu: .left)`

still not working

**Comment by jonkykong** [comment]
@shaharukhs I am unable to reproduce your problem in the demo project. Make sure you're following the README carefully and not setting the right side menu. Make sure you're using the latest code.

**Comment by shaharukhs** [comment]
Wait I will give you reproduced demo project.
It's happening only with iOS11 for iOS10 working fine.

**Comment by shaharukhs** [comment]
Please check attachment and run on both iOS 10 and iOS11.

iOS 10 working but for iOS 11 sidemenu opening from both side.

Thank you.

[sideMenu_iOS11.zip](https://github.com/jonkykong/SideMenu/files/1378266/sideMenu_iOS11.zip)

**Comment by jonkykong** [comment]
@shaharukhs:

1. You indicated you are using the latest version of SideMenu, however that is incorrect. The latest is 3.1.1. This project has 3.0.1. Run `pod update`.
2. You are adding both pan and edge gestures to the same view, which is redundant and can potentially conflict with each other. Don't do that. Pan gestures utilize the entire view so you don't need edge gestures. Otherwise, use only edge gestures. Please read more on the gesture documentation from Apple if you need more information.

**Comment by shaharukhs** [comment]
Yeah it was 3.0.1 don't know how because demo project created today itself.

Thank you for your help. Problem was solved with 3.1.1
